### DISCUSSION TITLE
Build fails: cannot open include file at llvm

### DISCUSSION THREAD

**Discussion by qir-at-tum** [intel/llvm#14104] [Q&A]
Hi, when trying to build Syclomatic, I'm getting the following error: "

`C:\Users\Ruili\workspace\SYCLomatic\llvm\include\llvm/SYCLLowerIR/DeviceConfigFile.hpp(33): fatal error C1083: Cannot open include file: 'llvm/SYCLLowerIR/DeviceConfigFile.inc': No such file or directory`

I'm working on a clean install, for the most part, and just don't know what is going on. Has anyone experienced this issue?

**Comment by ph0b** [comment]
I've experienced this issue when building some version of DPC++ with MSVC 2022 - fix was then https://github.com/intel/llvm/commit/b1c514a9df3b4e206a803fd63c794bc76c8413bc

**Comment by bader** [comment]
@qir-at-tum, please, open an issue in https://github.com/oneapi-src/SYCLomatic repository.
The commit @ph0b mentioned was done in April, so I assume similar dependency might be missing for some of SYCLomatic components.
